astronomer/providers/google/cloud/hooks/gcs.py

```diff
@@ -14,6 +15,17 @@
 class GCSHookAsync(GoogleBaseHookAsync):
     """GCSHookAsync run on the trigger worker, inherits from GoogleBaseHookAsync"""
 
+    def __init__(self, **kwargs: Any):
+        warnings.warn(
+            (
+                "This class is deprecated and will be removed in 2.0.0."
```

Comment:
Should we also add this to the docstring of the class for both the hook and triggers?

Reply:
yep, sounds good! just added